bigquery/google/cloud/bigquery/client.py

```diff
@@ -207,6 +208,20 @@ def get_dataset(self, dataset_ref):
             method='GET', path=dataset_ref.path)
         return Dataset.from_api_repr(api_response, self)
 
+    def get_table(self, table_ref):
+        """Fetch the table referenced by ``table_ref``
+
+        :type table_ref:
+            :class:`google.cloud.bigquery.table.TableReference`
+        :param table_ref: the table to use.
+
+        :rtype: :class:`google.cloud.bigquery.table.Table`
+        :returns: a ``Table`` instance
+        """
+        api_response = self._connection.api_request(
+            method='GET', path=table_ref.path)
+        return Table.from_api_repr(api_response, self)
```

Comment:
Maybe out of scope, but why are the `from_api_repr` methods public?

Reply:
Mostly because it was a convention that already existed in this library. I'm not sure why some classes had it and some don't.

It's relatively common for API representations be serialized to JSON. I know for certain the `bq` command-line tool takes "JSON files" as arguments for things like schemas, which map directly to the API representation, so it's not unheard of that someone would have something they want to parse.

The other thing I might like to use these methods for is to allow setting properties that the client libraries don't yet support (as they are introduced in alpha releases or just have trouble keeping up since this is a hand-written library)